utils/runtime-fuzzer/src/generator.rs

```diff
@@ -231,3 +231,13 @@ fn arbitrary_limited_bytes(u: &mut Unstructured, limit: usize) -> Result<Vec<u8>
     let arb_size = u.int_in_range(0..=limit)?;
     u.bytes(arb_size).map(|bytes| bytes.to_vec())
 }
+
+fn arbitrary_value(u: &mut Unstructured, current_balance: u128) -> Result<u128> {
+    let (lower, upper) = match u.int_in_range(0..=99)? {
+        0..=4 => (0, EXISTENTIAL_DEPOSIT),
+        5..=19 => (0, 0),
+        _ => (EXISTENTIAL_DEPOSIT, current_balance),
```

Comment:
if you send message wout "allow_death" in gear calls that you will fail on submitting in case of `0 < user value - arbitrary value < ED`

Reply:
double check it pls

Reply:
> double check it pls

all gear calls are sent with `keep_alive` being `false`, so it's always case of `AllowDeath`